tools/gdb_pretty_printer/nlohmann-json.py

```diff
@@ -1,7 +1,7 @@
 import gdb
 import re
 
-ns_pattern = re.compile(r'nlohmann::json_v(?P<v_major>\d+)_(?P<v_minor>\d+)_(?P<v_patch>\d+)(?P<tags>\w*)::(?P<name>.+)')
+ns_pattern = re.compile(r'nlohmann(::nlohmann_json(?P<tags>\w*))?(::v(?P<v_major>\d+)_(?P<v_minor>\d+)_(?P<v_patch>\d+))?::(?P<name>.+)')
```

Comment:
I think these changes do not belong to this PR.

Reply:
Without it, the GDB pretty printer isn't usable, but I can factor it out if you prefer.

Reply:
Oh sorry, I did not look at the code any only assumed that this was not meant for here.